webapp/channels/src/actions/schedule_message.ts

```diff
@@ -28,3 +32,25 @@ export function createSchedulePost(schedulePost: ScheduledPost): ActionFuncAsync
         }
     };
 }
+
+export default function fetchTeamScheduledPosts(teamId: string, includeDirectChannels: boolean): ActionFuncAsync<{data?: {[key: string]: ScheduledPost[]}; error?: string}, GlobalState> {
```

Comment:
I think this type is incorrect because `ActionFuncAsync` already assumes you're returning an object with `data` and `error` fields
```suggestion
export default function fetchTeamScheduledPosts(teamId: string, includeDirectChannels: boolean): ActionFuncAsync<{[key: string]: ScheduledPost[]}, GlobalState> {
```